Classes/Backend/TceMain.php

```diff
@@ -492,6 +492,7 @@ public function moveRecord_firstElementPostProcess($table, $uid, $destPid, $move
 
         // Move the immediate record, which may itself be a placeholder or an original record.
         $row['uid'] = $uid;
+        $row['colPos'] = $newColumnNumber;
```

Comment:
If you look at the code, it is hopefully clear that we should not be assigning any `colPos` value here in this function. All of that should take place in `ContentService->moveRecord`. The only property we set here is the `uid` and we only do so because this property doesn't exist in the input field array but Flux methods expect it to be there when, for example, you call `RecordService->update`.